### DISCUSSION TITLE
Help with Station selection issue.

### DISCUSSION THREAD

**Discussion by 19Slug79** [xspanger3770/GlobalQuake#326] [General]
Hello everyone. I'm having an issue on the station selection page. After I select the stations, There is no button to return to the main page to initiate Global quake. Any ideas? am I missing something?
<img width="1512" alt="Screenshot 2024-12-21 at 4 30 24 PM" src="https://github.com/user-attachments/assets/bf17f6b5-42a7-4607-a1c6-471b47f03095" />

**Comment by HaruKor** [comment]
Press the X button on the top left.
